### PR TITLE
Add verbose version flag to list dependencies as well as the version

### PR DESCRIPTION
**Pull request by zserge**
Add a `-vv` CLI flag to grafana-server to print the version _and_ the Go module dependencies that are _actually_ included into the final binary. Could be more useful that `go mod graph` or `go list -m all`.

### CODE REVIEW

File: pkg/cmd/grafana-server/main.go [hunk 1]
```diff
@@ -68,8 +70,16 @@ func main() {
 
 	flag.Parse()
 
-	if *v {
+	if *v || *vv {
 		fmt.Printf("Version %s (commit: %s, branch: %s)\n", version, commit, buildBranch)
+		if *vv {
+			fmt.Println("Dependnencies:")
```
Comment (aknuds1) [inline pkg/cmd/grafana-server/main.go:76]:
```suggestion
			fmt.Println("Dependencies:")
```

**Review by aknuds1 — APPROVED** [review]
LGTM, but please see my suggestion to fix a typo :)

**Comment by aknuds1** [comment]
Also set the correct PR milestone.

**Review by marefr — APPROVED** [review]
LGTM 

As I said earlier a mention of that this is available, perhaps in https://github.com/grafana/grafana/blob/main/UPGRADING_DEPENDENCIES.md#go-dependencies would be nice

**Comment by marefr** [comment]
@zserge can we get this merged or you plan to fix something more? Documentation as I suggested could potentially be handled in a separate PR.

**Pull request merged**